tower/Cargo.toml

```diff
@@ -20,10 +20,6 @@ edition = "2018"
 rust-version = "1.64.0"
 
 [features]
-
-# Internal
-__common = ["futures-core", "pin-project-lite"]
```

Comment:
If we are removing this, we probably want the removal to appear in the changelog, which may require this change to happen in a dedicated PR.

Reply:
Do people really enable features with `__`-prefixed names? It seems pretty clear that it's internal and may go away at any time..

Reply:
The double underscores is clear to me that it's not publicly stable.

Reply:
I think the decision to remove it can be postponed.